artichoke-backend/src/value.rs

```diff
@@ -395,7 +396,8 @@ impl Convert<Value, Value> for Artichoke {
 
 impl fmt::Display for Value {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{}", self.to_s())
+        let string_repr = String::from_utf8_lossy(&self.to_s()).to_string();
```

Comment:
ditto above it is not necessary to turn the `Cow` into an owned `String`

`String::from_utf8_lossy` only allocates a `String` if the byte slice was not UTF-8. By keeping the byte vector alive, we keep the `Cow::Borrowed` alive.

Assigning self.to_s() to a let binding allows us to avoid the `to_string`.

I also prefer calling `.as_slice()` instead of relying on deref coercion to turn `Vec`s and `String`s into slices because I find it to be more explicit and more clearly communicate intent.